widget_driver/example/pubspec.yaml

```diff
@@ -23,9 +23,11 @@ dev_dependencies:
     sdk: flutter
   flutter_lints: ^1.0.4
   mocktail: ^0.3.0
-  widget_driver_test: ^0.0.1
+  widget_driver_test:
+    path: ../../widget_driver_test
```

Comment:
I looked at other packages and they usually do the path-reference to their parent packages.
It makes sense since we are not publishing the example app.

But we have one small issue here. Since we have so many packages which we depend on :)

I could make `widget_driver_test` and `widget_driver_generator` to be imported by `path`.
But for `widget_driver` I need to keep the real version import..
This is because the `widget_driver_test` has an internal dependency to the real imported version of `widget_driver`... And then flutter pub get was not able to figure out which version to get..

So I think we can keep it like this.

Maybe in the future we can fix this differently by having the `widget_driver_test` not depend directly on `widget_driver` but instead have some dependency free package which it can import types from. But I think for now this works.